apps/testapp/src/context/CBWSDKProvider.tsx

```diff
@@ -27,27 +28,41 @@ const dynamicallyImportSDK = (version: SDKVersionType) => {
 };
 
 export function CBWSDKProvider({ children }: CBWSDKProviderProps) {
-  const [version, setVersion] = React.useState<SDKVersionType>('master');
+  const [version, setVersion] = React.useState<SDKVersionType | undefined>(undefined);
   const [sdk, setSdk] = React.useState(null);
   const [provider, setProvider] = React.useState(null);
 
+  useEffect(() => {
+    if (version === undefined) {
+      const savedVersion = localStorage.getItem(SELECTED_SDK_KEY) as SDKVersionType;
+      setVersion(sdkVersions.includes(savedVersion) ? (savedVersion as SDKVersionType) : 'master');
+    }
+  }, [version]);
+
   useEffect(() => {
     const selectedSDK = dynamicallyImportSDK(version);
-    const cbwsdk = new selectedSDK({
-      appName: 'Test App',
-      enableMobileWalletLink: true, // beta feature
-    });
-    setSdk(cbwsdk);
-    const cbwprovider = cbwsdk.makeWeb3Provider('http');
-    setProvider(cbwprovider);
+    if (selectedSDK) {
+      const cbwsdk = new selectedSDK({
+        appName: 'Test App',
+        enableMobileWalletLink: true, // beta feature
+      });
+      setSdk(cbwsdk);
+      const cbwprovider = cbwsdk.makeWeb3Provider('http');
+      setProvider(cbwprovider);
+    }
   }, [version]);
 
+  const setSDKVersion = (version: SDKVersionType) => {
+    localStorage.setItem(SELECTED_SDK_KEY, version);
+    setVersion(version);
+  };
+
```

Comment:
save selected sdkversion in localstorage, so sdk remians the same version on each refresh